### PR TITLE
Removes more repeated Pokemon from routes

### PR DESCRIPTION
**Pull request by osherwott**
* Poliwag from Johto Route 31 water
* Wingull from Sinnoh Route 213 water
* Dunsparce from Kalos Route 3 land
* Machoke from South Lake Miloch weather (Overcast, Snow, Blizzard, Sunny, and Sandstorm)
* Tyrogue from South Lake Miloch weather (Overcast)
* Drifloon from South Lake Miloch weather (Fog)
* Pyukumuku from South Lake Miloch weather (Thunderstorm)
* Jellicent from Lake of Outrage Thunderstorm
* Mareanie from Motostoke Riverbank weather (Thunderstorm)
* Hawlucha from Hammerlocke Hills weather (Overcast)
* Greedent from Giant's Bed weather (Blizzard)
* Beartic from Frigid Sea Blizzard
* Skwovet from Ballimere Lake land

### CODE REVIEW

**Comment by osherwott** [comment]
Forgot to link to/mention #3351

**Comment by xslk** [comment]
Weather pokemon are deliberate

**Comment by Qwertypop04** [comment]
> Weather pokemon are deliberate 

They are not deliberate

**Comment by osherwott** [comment]
Noted and fixed

**Comment by osherwott** [comment]
Should any edits be needed, feel free to do so. The "Allow edits by maintainers" box is ticked.

**Comment by Qwertypop04** [comment]
> Noted and fixed

SLK was incorrect, the repeats on weather routes were not intentional.

**Comment by xslk** [comment]
Qwerty cleared it up. Mobile view = L

**Comment by osherwott** [comment]
Noted and fixed... again!

File: src/scripts/wildBattle/RouteData.ts [hunk 1]
```diff
@@ -3231,7 +3231,7 @@ Routes.add(new RegionRoute(
 Routes.add(new RegionRoute(
     'Ballimere Lake', GameConstants.Region.galar, 53,
     new RoutePokemon({
-        land: ['Boltund', 'Audino', 'Skwovet', 'Yamper', 'Mime Jr.', 'Aron'],
+        land: ['Boltund', 'Audino', 'Yamper', 'Mime Jr.', 'Aron'],
```
Comment (Jaaslet) [inline src/scripts/wildBattle/RouteData.ts:3234]: This looks like a mistake?

File: src/scripts/wildBattle/RouteData.ts [hunk 2]
```diff
@@ -3127,7 +3127,7 @@ Routes.add(new RegionRoute(
           new SpecialRoutePokemon(['Araquanid', 'Dewpider', 'Nidoking', 'Nidoqueen', 'Obstagoon', 'Vaporeon', 'Snorlax'],  new WeatherRequirement([WeatherType.Rain])),
           new SpecialRoutePokemon(['Araquanid', 'Galvantula', 'Electabuzz', 'Dewpider', 'Elekid', 'Electivire', 'Jolteon', 'Nidoking', 'Snorlax'],  new WeatherRequirement([WeatherType.Thunderstorm])),
           new SpecialRoutePokemon(['Snorunt', 'Cryogonal', 'Vanillish', 'Vanillite', 'Nidoking', 'Nidoqueen', 'Vanilluxe', 'Espeon', 'Glalie', 'Froslass', 'Snorlax'],  new WeatherRequirement([WeatherType.Snow])),
-          new SpecialRoutePokemon(['Snorunt', 'Cryogonal', 'Vanillish', 'Absol', 'Vanillite', 'Glalie', 'Froslass', 'Glaceon', 'Greedent', 'Nidoking', 'Nidoqueen', 'Vanilluxe', 'Snorlax'],  new WeatherRequirement([WeatherType.Blizzard])),
+          new SpecialRoutePokemon(['Snorunt', 'Cryogonal', 'Vanillish', 'Absol', 'Vanillite', 'Glalie', 'Froslass', 'Glaceon', 'Nidoking', 'Nidoqueen', 'Vanilluxe', 'Snorlax'],  new WeatherRequirement([WeatherType.Blizzard])),
```
Comment (Jaaslet) [inline src/scripts/wildBattle/RouteData.ts:3130]: This looks like a mistake?

**Review by Jaaslet — APPROVED** [review]
I found two places where it looks like it was not a duplicate (or i am blind). But other than that, it looks good

**Review by Jaaslet — CHANGES_REQUESTED** [review]
Approved by mistake...

**Review by Jaaslet — APPROVED** [review]
Someone pointed out the mons i thought was a mistake is below special, in the headbutt section...
Thanks for the PR!

**Pull request merged**